tap_linkedin_ads/schema.py

```diff
@@ -75,12 +75,16 @@ def get_schemas():
         )
 
         # Add additional metadata
+        mdata_map = metadata.to_map(mdata)
         if stream_name in ('ad_analytics_by_campaign', 'ad_analytics_by_creative'):
-            mdata_map = metadata.to_map(mdata)
             mdata_map[('properties', 'date_range')]['inclusion'] = 'automatic'
             mdata_map[('properties', 'pivot')]['inclusion'] = 'automatic'
             mdata_map[('properties', 'pivot_value')]['inclusion'] = 'automatic'
-            mdata = metadata.to_list(mdata_map)
+
+        if stream_name == "accounts":
+            mdata_map[('properties', 'reference_organization_id')]['inclusion'] = 'automatic'
```

Comment:
@dbshah1212 Please add the integration test for this code change once the tap tester coverage (https://github.com/singer-io/tap-linkedin-ads/pull/30) is merged